src/error_estimation/exact_solution.C

```diff
@@ -469,12 +469,19 @@ void ExactSolution::_compute_error(std::string_view sys_name,
     _equation_systems_fine->get_system(sys_name) :
     _equation_systems.get_system (sys_name);
 
+  const MeshBase & mesh = computed_system.get_mesh();
+
   const Real time = _equation_systems.get_system(sys_name).time;
 
   const unsigned int sys_num = computed_system.number();
   const unsigned int var = computed_system.variable_number(unknown_name);
-  const unsigned int var_component =
-    computed_system.variable_scalar_number(var, 0);
+  unsigned int var_component = 0;
+  for (const auto var_num : make_range(cast_int<decltype(var)>(0), var))
```

Comment:
```suggestion
  for (const auto var_num : make_range(var))
```
I don't know why I added the unnecessary `begin` argument 🙁